src/storage/file_io.c

```diff
@@ -11873,3 +11875,136 @@ fileio_is_formatted_page (THREAD_ENTRY * thread_p, const char *io_page)
   free_and_init (ref_page);
   return is_formatted;
 }
+
+/*
+ * fileio_lob_remove_dir () - Remove LOB directory(OS level).
+ *
+ * return	 : Error code.
+ * path_key (in) : Path key for the LOB directory (keyword or path information).
+ *
+ * NOTE: This function receives the path information for a LOB directory
+ *       and recursively deletes the corresponding physical directory at the OS level.
+ *       The input parameter 'path_key' can take two forms:
+ *       1. keyword: If a keyword is provided (e.g., "ces" or "hfid"..),
+ *          all directories in the LOB directory that include the keyword will be deleted.
+ *       2. lob directory name: If a directory name under the LOB directory is provided,
+ *          the function will traverse and delete that specific directory recursively.
+ */
+int
+fileio_lob_remove_dir (char *path_key)
+{
+#if defined(SERVER_MODE) || defined(SA_MODE)
+  DIR *dir_p;
+  struct dirent *dir_entry;
+  struct stat statbuf;
+  char base_dir[PATH_MAX], full_path[PATH_MAX], re_path[PATH_MAX];
+  char *pos, *keyword;
+  int result, sub_result;
+  result = sub_result = 0;
+  int error = NO_ERROR;
+
+  if (stat (es_base_dir, &statbuf) != 0 || !S_ISDIR (statbuf.st_mode))
+    {
+      error = ER_ES_NO_LOB_PATH;
+      return error;
+    }
+  dir_p = opendir (es_base_dir);
+  if (dir_p == NULL)
+    {
+      error = ER_ES_NO_LOB_PATH;
+      return error;
+    }
+  closedir (dir_p);
+
+  pos = strrchr (path_key, '/');
```

Comment:
이 부분 외에도 치준님이 작성한 부분에 모두 적용

```suggestion
  pos = strrchr (path_key, PATH_SEPARATOR);
```

Reply:
반영했습니다.